### PR TITLE
OCPP 2.0 Enums and helper methods

### PR DESCRIPTION
**Pull request by tropxy**

### CODE REVIEW

**Comment by tropxy** [comment]
Yes, my intention was just to start discussing it with something. I am aware this is not enough, but we have to start from somewhere

File: ocpp/v20/helpers.py [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    @property
+    def validfrom(self):
+        return self.chargingprofile['validFrom']
+
+    @validfrom.setter
+    def validfrom(self, value=datetime.utcnow().isoformat()):
+        self.chargingprofile['validFrom'] = value
+
+    @property
+    def validto(self):
```
Comment (laysauchoa) [inline ocpp/v20/helpers.py]: I thought this was a typo `validto`

File: ocpp/v20/helpers.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+        """
+
+        Args:
+            chargingscheduleperiod (ChargingSchedulePeriodConstructor):
+            List of ChargingSchedulePeriod elements defining maximum power or
+            current usage over time.
+            duration (int): Duration of the charging schedule in seconds.
+            If the duration is left empty, the last period will continue
+            indefinitely
+            startschedule (DateTime): Starting point of an absolute schedule.
```
Comment (laysauchoa) [inline ocpp/v20/helpers.py]: is not datetime here? or datetime.datetime type?

**Comment by lsaavedr** [comment]
related to #61

**Comment by lsaavedr** [comment]
Why subclassing from Enum is not needed (I need it in my django backend), using getattr is not a good practice 😭, Enum was proposal for this!

**Comment by OrangeTux** [comment]
@lsaavedr I commented in #61 to keep discussion in one 1 place.

**Comment by Jared-Newell-Mobility** [comment]
Due to the age and that 2.1 is to be used in place of 2.0, I'll close this PR

**Pull request closed without merging**